contracts/oracle/FastPriceFeed.sol

```diff
@@ -212,10 +261,28 @@ contract FastPriceFeed is ISecondaryPriceFeed, IFastPriceFeed, Governable {
         _setPricesWithBits(_priceBits, _timestamp);
     }
 
-    function setPricesWithBitsAndExecute(uint256 _priceBits, uint256 _timestamp, uint256 _endIndexForIncreasePositions, uint256 _endIndexForDecreasePositions) external onlyUpdater {
+    function setPricesWithBitsAndExecute(
+        uint256 _priceBits,
+        uint256 _timestamp,
+        uint256 _endIndexForIncreasePositions,
+        uint256 _endIndexForDecreasePositions,
+        uint256 _maxIncreasePositions,
+        uint256 _maxDecreasePositions
```

Comment:
do we need to pass _maxIncreasePositions/_maxDecreasePositions each time? or it can be stored in contract variable? i m thinking about gas, keeper calls that method pretty often

or maybe better: what about passing last 5 args as 1 uint256?